### PR TITLE
enhancement/venn-diagram-onclick/#9

### PR DESCRIPTION
**Pull request by alexkirsz**
#9

### CODE REVIEW

File: src/components/tooltip/hover-tooltip.jsx [hunk 1]
```diff
@@ -19,16 +19,18 @@ var HoverTooltip = React.createClass({
   },
 
   render() {
+    let {hasTooltip, renderTooltip, children, ...otherProps} = this.props;
+
     return (
-      <div onMouseMove={this._handleMouseMove}>
-        {this.props.hasTooltip &&
+      <div {...otherProps} onMouseMove={this._handleMouseMove}>
+        {hasTooltip &&
```
Comment (zallek) [inline src/components/tooltip/hover-tooltip.jsx:26]: was does this mean ?

File: src/components/venn-diagram/venn-canvas.jsx [hunk 2]
```diff
@@ -17,25 +24,23 @@ const VennCanvas = React.createClass({
...
 
   /**
    * Recalculate the venn diagram so that it fits into the canvas
    */
   _scale() {
-    this.setState({
-      width: this.getDOMNode().offsetWidth,
-      height: this.getDOMNode().offsetHeight
-    });
+    let {offsetWidth, offsetHeight} = this.getDOMNode();
```
Comment (zallek) [inline src/components/venn-diagram/venn-canvas.jsx:35]: you will have to do that in every chart component right ?

File: src/components/tooltip/hover-tooltip.jsx (hunk 1 above)
Comment (alexkirsz) [inline src/components/tooltip/hover-tooltip.jsx:26]: This is a recommended pattern http://facebook.github.io/react/docs/transferring-props.html

File: src/components/venn-diagram/venn-canvas.jsx (hunk 2 above)
Comment (alexkirsz) [inline src/components/venn-diagram/venn-canvas.jsx:35]: `width` and `height` are used to scale the venn solution (see https://github.com/botify-labs/react-components/blob/enhancement/venn-diagram-onclick/%239/src/components/venn-diagram/venn-canvas.jsx#L71). Google Charts scale themselves automatically.

Comment (zallek) [inline src/components/venn-diagram/venn-canvas.jsx:35]: ok

**Comment by zallek** [comment]
:+1:

**Pull request merged**